src/help/guides/extension-development/model-reference.md

```diff
@@ -0,0 +1,267 @@
+The Manga object contains most metadata about a manga. Different manga objects are differentiated by the Manga ID, and
+changing the Manga ID will return a different manga.
+
+### Required Fields
+
+| Name | Type | Description |
+|------|------|-------------|
+| `id` | String | The given identifier of this Manga. This may be unique to the source which uses it. For example, one source may use the value '1234' to identify a manga, whereas another one may use the value 'One-Piece' to identify |
+| `image` | String | A URL pointing to a thumbnail which can be displayed to present the manga. |
+| `rating` | Number | The rating which users have given this manga. Use `0` if the manga is unrated or the source has no rating system in place. |
+| `status` | [MangaStatus](#mangastatus) | The status of the manga, indicating if the manga is ongoing or completed. |
+| `titles` | String Array | An array of titles which this Manga is called. The first title is the manga's official title, while extra entries are treated as alternate titles for the manga. |
+
+### Optional Fields
+
+| Name | Type | Description |
+|------|------|-------------|
+| `langFlag` | String | A language code for the Manga, if one is available. Examples: `en` is English, `jp` is Japanese, etc. |
+| `langName` | String | The full name of the manga's language. `English`, `Japanese`, etc. |
+| `artist` | String | The name of the manga's artist. Multiple artists should be concatenated into a String with a delimiter such as a comma separating each artist. |
+| `author` | String | The name of the manga's author. Multiple authors should be concatenated into a String with a delimiter such as a comma separating each author. |
+| `avgRating` | Number | The average rating of the manga. May be the same value as the `rating` field. |
+| `covers` | String Array | Additional covers for the manga. Each entry is a URL pointing to the cover image. |
+| `desc` | String | A description of this manga. |
+| `follows` | Number | The number of followers of the manga if the source supports following mangas and displays follower counts. |
+| `tags` | [TagSection](#tagsection) Array | An array containing the various tag sections. Only the first entry is actually displayed in the app. |
+| `users` | Number | The number of users subscribed to the manga. May be the same value as the `follows` field. |
+| `views` | Number | The number of views this manga has had. |
+| `hentai` | Boolean | Whether or not this manga is a hentai. If it is set to true, the manga will not be visible to the user unless the user enables mature manga. |
+| `relatedIds` | String Array | The IDs of related manga. |
+| `lastUpdate` | String | A string showing when the manga was last updated. It might either be an absolute date (`March 1, 2021`) or a relative date (`2 days ago`). |
+
+## MangaStatus
+
+An integer enum representing the various statuses a manga can have.
+
+### Values
+
+| Name | Value | Description |
+|------|-------|-------------|
```

Comment:
As the column contains only integers, what would you think of centering the value? That would be inconsistent with the rest of the page so I'm not sure if its a good idea.
```suggestion
|------|:-----:|-------------|
```

Reply:
There's no point, since the number column is very tiny relatively.

Reply:
I like the idea of centering it. It's a solid style guideline.

Reply:
But why this column? In that case, shouldn't it apply to all enums?

Reply:
It should apply to columns which __only__ have integers. If the enumeration mapping is to strings or some other type, this wouldn't fit.

Reply:
I guess it makes sense with numbers.